script-test/playbackstrategies/modifiers/html5tests.js

```diff
@@ -0,0 +1,3260 @@
+require(
+  [
+    'squire',
+    'bigscreenplayer/playbackstrategy/modifiers/mediaplayerbase'
+  ],
+    function (Squire, MediaPlayerBase) {
+      describe('HTML5 Base', function () {
+        var injector = new Squire();
+        var html5Player;
+        var sourceContainer;
+        var player;
+        var mockSourceElement;
+        var mockVideoMediaElement;
+        var mockAudioMediaElement;
+        var metaDataCallback;
+        var finishedBufferingCallback;
+        var errorCallback;
+        var endedCallback;
+        var timeupdateCallback;
+        var waitingCallback;
+
+        var recentEvents = [];
+
+        var logger = jasmine.createSpyObj('logger', ['warn', 'debug', 'error']);
+
+        mockSourceElement = document.createElement('source');
+        mockVideoMediaElement = document.createElement('video');
+        mockAudioMediaElement = document.createElement('audio');
+
+        function playFromNearCurrentTime (currentTime, targetTime) {
```

Comment:
This function is only used once - should it just be part of the test that uses it instead?